lib/auth-client.js

```diff
@@ -158,8 +162,26 @@ AuthClient.prototype.getAuthUrl = function(options) {
   }
 
   if (options.singleSelect !== undefined) {
-    // eslint-disable-next-line camelcase
-    parameters.single_select = options.singleSelect === true;
+    if (typeof options.singleSelect === 'object'
+      && options.singleSelect !== null) {
```

Comment:
options.singleSelect = {}
options.singleSelect !== null (this gives true). I don't think it is possible for an object to be null